rust/otel-arrow-rust/benches/attribute_transform.rs

```diff
@@ -61,11 +61,42 @@ fn generate_dict_keys_attribute_batch(
 }
 
 fn bench_transform_attributes(c: &mut Criterion) {
+    // Pre-create AttributesTransform instances to avoid measuring their creation cost
+    let single_replace_no_delete = AttributesTransform {
```

Comment:
I had to extract the `AttributeTransform` instance outside of the benchmark to not include `AttributesTransform` creation time in the benchmark measurements.